daphne/src/lib.rs

```diff
@@ -126,6 +126,11 @@ pub enum DapAbort {
     #[error("invalidBatchInterval")]
     InvalidBatchInterval,
 
+    /// Invalid DAP version. Sent in response to requests that use a DAP version URL endpoint
+    /// that does not match to the version set in TaskConfig or simply does not exist.
+    #[error("InvalidProtocolVersion")]
```

Comment:
```suggestion
    #[error("invalidProtocolVersion")]
```

Reply:
Thanks for pointing this out! Completely missed this